### PR TITLE
Split preparing and testing into different tasks

### LINKED ISSUE #106: Split preparation into before_script on travis
The preparation is not required to be visible:

```
German phpBB Translation > clean:

[delete] Directory /home/travis/build/phpbb-de/phpbb-translation/packages does not exist or is not a directory.

German phpBB Translation > prepare:

[mkdir] Created dir: /home/travis/build/phpbb-de/phpbb-translation/packages

[mkdir] Created dir: /home/travis/build/phpbb-de/phpbb-translation/packages/de

[mkdir] Created dir: /home/travis/build/phpbb-de/phpbb-translation/packages/de_x_sie

[mkdir] Created dir: /home/travis/build/phpbb-de/phpbb-translation/packages/en

[mkdir] Created dir: /home/travis/build/phpbb-de/phpbb-translation/packages/upload

[mkdir] Created dir: /home/travis/build/phpbb-de/phpbb-translation/packages/git

German phpBB Translation > package:

[echo] Packaging HEAD

[phingcall] Calling Buildfile '/home/travis/build/phpbb-de/phpbb-translation/build.xml' with target 'git-archive'

German phpBB Translation > git-archive:

[echo] Getting archive for HEAD

[unzip] Extracting zip: /home/travis/build/phpbb-de/phpbb-translation/packages/git/HEAD.zip to /home/travis/build/phpbb-de/phpbb-translation/packages/git/HEAD

[phingcall] Calling Buildfile '/home/travis/build/phpbb-de/phpbb-translation/build.xml' with target 'prepare-package-language'

German phpBB Translation > prepare-package-language:

[echo] Preparing package (en)

[copy] Created 6 empty directories in /home/travis/build/phpbb-de/phpbb-translation/packages/en

[copy] Copying 120 files to /home/travis/build/phpbb-de/phpbb-translation/packages/en

[copy] Copying 3 files to /home/travis/build/phpbb-de/phpbb-translation/packages/en/language/en

[phingcall] Calling Buildfile '/home/travis/build/phpbb-de/phpbb-translation/build.xml' with target 'prepare-package-language'

German phpBB Translation > prepare-package-language:

[echo] Preparing package (de)

[copy] Created 6 empty directories in /home/travis/build/phpbb-de/phpbb-translation/packages/de

[copy] Copying 121 files to /home/travis/build/phpbb-de/phpbb-translation/packages/de

[copy] Copying 3 files to /home/travis/build/phpbb-de/phpbb-translation/packages/de/language/de

[phingcall] Calling Buildfile '/home/travis/build/phpbb-de/phpbb-translation/build.xml' with target 'create-package-language'

German phpBB Translation > create-package-language:

[echo] Creating archives (de)

[zip] Building zip: /home/travis/build/phpbb-de/phpbb-translation/packages/upload/lang_de.zip

[tar] Building tar: /home/travis/build/phpbb-de/phpbb-translation/packages/upload/lang_de.tar.gz
```

### PR DESCRIPTION
**Pull request by nickvergessen**
Fix #106

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -6,14 +6,16 @@
 	<property name="language" value="${language}" />
 
 	<!-- These are the main targets which you will probably want to use -->
-	<target name="all" depends="lint,clean,prepare,package,validate" />
-	<target name="build" depends="lint,clean,prepare,package" />
+	<target name="all" depends="prepare-structure,prepare,test,package" />
+	<target name="build" depends="prepare-structure,prepare,package" />
+	<target name="prepare_test" depends="prepare-structure,prepare" />
```
Comment (bantu) [inline build.xml]: prepare_test -> prepare-test

File: build.xml [hunk 2]
```diff
@@ -22,20 +24,17 @@
 		<mkdir dir="packages/git" />
 	</target>
 
-	<target name="package">
-		<echo msg="Packaging ${version}" />
+	<target name="prepare">
+		<echo msg="Exracting ${version}" />
```
Comment (bantu) [inline build.xml]: Extracting

File: build.xml (hunk 1 above)
Comment (nickvergessen) [inline build.xml]: Done

File: build.xml (hunk 2 above)
Comment (nickvergessen) [inline build.xml]: Done

**Pull request merged**